### ISSUE TITLE
docopt_macros is broken with rustc 1.9.0-dev (a9f34c86a 2016-04-06)

### ISSUE THREAD

**Issue by alsam** [docopt/docopt.rs#174]
Hello,

After upgrading rust to '1.9.0-dev (a9f34c86a 2016-04-06)' I've got the following syntax error:

```rust
/home/asamoilo/.cargo/registry/src/github.com-88ac128001ac3a9a/docopt_macros-0.6.82/src/macro.rs:141:32: 141:49 error: `ast::StructField_` does not name a structure [E0422]
/home/asamoilo/.cargo/registry/src/github.com-88ac128001ac3a9a/docopt_macros-0.6.82/src/macro.rs:141         codemap::dummy_spanned(ast::StructField_ {
                                                                                                                                    ^~~~~~~~~~~~~~~~~
/home/asamoilo/.cargo/registry/src/github.com-88ac128001ac3a9a/docopt_macros-0.6.82/src/macro.rs:141:32: 141:49 help: run `rustc --explain E0422` to see a detailed explanation
/home/asamoilo/.cargo/registry/src/github.com-88ac128001ac3a9a/docopt_macros-0.6.82/src/macro.rs:142:19: 142:34 error: unresolved name `ast::NamedField` [E0425]
/home/asamoilo/.cargo/registry/src/github.com-88ac128001ac3a9a/docopt_macros-0.6.82/src/macro.rs:142             kind: ast::NamedField(ident(name), ast::Visibility::Public),
                                                                                                                       ^~~~~~~~~~~~~~~
/home/asamoilo/.cargo/registry/src/github.com-88ac128001ac3a9a/docopt_macros-0.6.82/src/macro.rs:142:19: 142:34 help: run `rustc --explain E0425` to see a detailed explanation
/home/asamoilo/.cargo/registry/src/github.com-88ac128001ac3a9a/docopt_macros-0.6.82/src/macro.rs:141:9: 146:11 error: mismatched types:
 expected `syntax::ast::StructField`,
    found `syntax::codemap::Spanned<_>`
(expected struct `syntax::ast::StructField`,
    found struct `syntax::codemap::Spanned`) [E0308]
/home/asamoilo/.cargo/registry/src/github.com-88ac128001ac3a9a/docopt_macros-0.6.82/src/macro.rs:141         codemap::dummy_spanned(ast::StructField_ {
/home/asamoilo/.cargo/registry/src/github.com-88ac128001ac3a9a/docopt_macros-0.6.82/src/macro.rs:142             kind: ast::NamedField(ident(name), ast::Visibility::Public),
/home/asamoilo/.cargo/registry/src/github.com-88ac128001ac3a9a/docopt_macros-0.6.82/src/macro.rs:143             id: ast::DUMMY_NODE_ID,
/home/asamoilo/.cargo/registry/src/github.com-88ac128001ac3a9a/docopt_macros-0.6.82/src/macro.rs:144             ty: ty,
/home/asamoilo/.cargo/registry/src/github.com-88ac128001ac3a9a/docopt_macros-0.6.82/src/macro.rs:145             attrs: vec!(),
/home/asamoilo/.cargo/registry/src/github.com-88ac128001ac3a9a/docopt_macros-0.6.82/src/macro.rs:146         })
/home/asamoilo/.cargo/registry/src/github.com-88ac128001ac3a9a/docopt_macros-0.6.82/src/macro.rs:141:9: 146:11 help: run `rustc --explain E0308` to see a detailed explanation
error: aborting due to previous error
```
Thanks

**Comment by BurntSushi** [comment]
Fixed in 928c0e84014d54b5451ed2aadc7f39b3fa87e50f. It's on crates.io in `docopt_macros 0.6.83`. Thanks for the report!

**Comment by alsam** [comment]
Thanks!
